### PR TITLE
Fix: M2-3023 - Builder - Items Flow: Placeholder text isn't displayed in empty fields

### PR DESCRIPTION
**Pull request by muhurov**

### CODE REVIEW

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlowContent/ItemFlowContent.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import { Condition } from 'shared/state';
+import { StyledFlexColumn, StyledBodyLarge, theme, variables } from 'shared/styles';
+import { ConditionRow } from 'modules/Builder/components';
+
+import { ItemFlowContentProps } from './ItemFlowContent.types';
+import { SummaryRow } from '../SummaryRow';
+
+export const ItemFlowContent = ({
+  items,
```
Comment (IhorNesterenko99) [inline src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlowContent/ItemFlowContent.tsx:12]:
```suggestion
  conditions,
```

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx [hunk 2]
```diff
@@ -2,41 +2,20 @@ import { useTranslation } from 'react-i18next';
 import { useFormContext, useFieldArray } from 'react-hook-form';
 import { Badge } from '@mui/material';
 
-import { ConditionRow, ToggleItemContainer } from 'modules/Builder/components';
-import { StyledFlexColumn } from 'shared/styles';
-import { Condition } from 'shared/state';
+import { ToggleItemContainer } from 'modules/Builder/components';
 
 import { Actions } from './Actions';
```
Comment (IhorNesterenko99) [inline src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx:7]:
```suggestion
import { FlowActions } from './FlowActions';
```

**Review by IhorNesterenko99 — APPROVED** [review]

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlowContent/ItemFlowContent.tsx (hunk 1 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlowContent/ItemFlowContent.tsx]: done

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx (hunk 2 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx]: done

**Review by kdanilenka32 — APPROVED** [review]

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx [hunk 3]
```diff
@@ -68,21 +46,21 @@ export const ItemFlow = ({ name, index, onRemove }: ItemFlowProps) => {
...
         onRemove: handleRemoveCondition,
-        conditionalError,
+        'data-testid': dataTestid,
       }}
       headerContentProps={{
         name: itemName,
         onAdd: handleAddCondition,
         onRemove,
-        'data-testid': `builder-activity-item-flow-${index}`,
+        'data-testid': dataTestid,
```
Comment (alex-i-e) [inline src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx:61]: does it make sense?
```suggestion
        'data-testid': `${dataTestid}-header`,
```

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx [hunk 4]
```diff
@@ -68,21 +46,21 @@ export const ItemFlow = ({ name, index, onRemove }: ItemFlowProps) => {
...
-      HeaderContent={Actions}
+      Content={ItemFlowContent}
+      HeaderContent={ItemFlowActions}
       contentProps={{
-        items: conditions,
         name: itemName,
+        conditions,
         onRemove: handleRemoveCondition,
-        conditionalError,
+        'data-testid': dataTestid,
```
Comment (alex-i-e) [inline src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx:55]: 'data-testid': dataTestid,
        'data-testid': `${dataTestid}-content`,

**Review by alex-i-e — APPROVED** [review]

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx (hunk 3 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx:61]: it is used just for the actions like Add/Remove/Collapse so there is no sense to separate header and content

File: src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx (hunk 4 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlow.tsx:55]: it is used just for the actions like Add/Remove/Collapse so there is no sense to separate header and content

**Pull request merged**